src/mame/bfm/bfm_blackbox.h

```diff
@@ -0,0 +1,299 @@
+// license:BSD-3-Clause
+// copyright-holders:SomeRandomGuyIdk
+/**********************************************************************
+
+	Bellfruit Black Box
+
+**********************************************************************/
+
+#ifndef MAME_INCLUDES_BFM_BLACKBOX_H
+#define MAME_INCLUDES_BFM_BLACKBOX_H
```

Comment:
This doesn’t match the file path.  Also, is there a good reason for having this stuff in a header?  Is it going to be inherited from another source file?  If not, the declarations should be in the .cpp file, and everything besides the driver definitions should be in an anonymous namespace.

Reply:
Fixed. I just did it to split things up since the driver has a lot of classes, and will probably end up getting a few more once I add the stepper games.